forgerock-openbanking-uk-aspsp-as/forgerock-openbanking-uk-aspsp-as-gateway/forgerock-openbanking-uk-aspsp-as-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/as/api/authorisation/AuthorisationApiControllerTest.java

```diff
@@ -62,96 +70,126 @@
     private AMGatewayService amGatewayService;
     @Mock
     private DiscoveryConfig discoveryConfig;
-    @InjectMocks
-    private AuthorisationApiController authorisationApiController;
+    @Mock
+    private JwtOverridingService jwtOverridingService;
+    @Mock
+    private TokenUsageService tokenUsageService;
+
+    private AuthorisationApiController authorisationApiController ;
+
+    @Before
+    public void setUp() throws Exception{
+        this.authorisationApiController = new AuthorisationApiController(this.headLessAuthorisationService,
+                this.cryptoApiClient, this.tppStoreService, "cookieName", false, this.amGatewayService,
+                this.jwtOverridingService, this.tokenUsageService, this.discoveryConfig);
+    }
 
     @Test
-    public void shouldNotThrowExceptionWhenContainAllScopes() throws OBErrorException, OBErrorResponseException {
+    public void shouldNotThrowExceptionWhenContainAllScopes() throws OBErrorException, OBErrorResponseException,
```

Comment:
This is a bit negative ;-)

Normally we would just call this `shouldGetAuthorisationGivenAllScopes()`

Reply:
I just couldn't face renaming all these tests :-D